packages/require/src/require.js

```diff
@@ -59,8 +59,8 @@ export function require(urlOrSource, transform, _readFile=readFileWeb, _base, ro
 
     if (isModule) {
       readFile = readModule
-      base = root = url + '/'
     }
+    base = url
```

Comment:
not sure if this will break something else in that long line of imports of jscad/io. I will check it out again with this line included.

Reply:
looks it works after disabling jscadio bundle, al hose funky imports are ok.